docs/deployment/getting_started_locally.md

```diff
@@ -198,6 +198,9 @@ Hence, if you want to access the shoot cluster, you have to run the following co
 cat <<EOF | sudo tee -a /etc/hosts
 
 # Begin of Gardener local setup section
+# Garden Cluster domain
+127.0.0.1 garden.local.gardener.cloud
```

Comment:
Can you explain why we need this entry in the local `/etc/hosts/` file?

Reply:
Since skaffold is responsible for pushing images from our host machine to the in-cluster registry, we want `garden.local.gardener.cloud` to resolve to `localhost`.

Reply:
As discussed offline, the entry is required for the local setup of Lakom (https://github.com/gardener/gardener-extension-shoot-lakom-service/pull/93) and it's not very understandable having this in the section of "Accessing the `Shoot` Cluster".

In addition, Lakom might be an exception here because, unlike other/ordinary workload, the extension doesn't access the registry via `containerd` for which we have a proper `hosts.toml` configuration (mapping the `localhost:5001` host namespace to `garden.local.gardener.cloud:5001`) in place.

Maybe it's best to document this use-case and action item in the Lakom repo accordingly.